src/blocks/menu/__link/menu__link.css

```diff
@@ -0,0 +1,39 @@
+.menu__link {
+  display: inline-block;
+  display: block;
+  padding: 15px;
+  text-decoration: none;
+  color: rgb(255, 255, 255);
+  font-weight: 800;
+  text-transform: uppercase;
+  margin: 0 10px;
+  transition: all 0.5s;
+  position: relative;
+}
+
+.menu__link::before {
+  transition: all 0.5s;
+}
+
+.menu__link::after {
+  transition: all 0.5s;
+
```

Comment:
it's empty line, installed for your IDE PRETTIER [https://www.digitalocean.com/community/tutorials/how-to-format-code-with-prettier-in-visual-studio-code-ru](url)